packages/testsuite/cypress/support/form-editing.ts

```diff
@@ -1,7 +1,23 @@
 Cypress.Commands.add("editForm", (formId) => {
-  const editButton = "#" + formId + ' a.clickable[data-operation="edit"';
+  const editButton = "#" + formId + ' a.clickable[data-operation="edit"]';
   cy.get(`#${formId}-editing`).should("not.be.visible");
   cy.get(editButton).click();
+  // Workauround - sometimes is required to click more times, especialy on overloaded system.
```

Comment:
> sometimes is required to click more times, especialy on overloaded system.

-> the form is sometimes not loaded in time and Cypress is not able to recover from such state and click the button.

-> The while cycle is alwas 5 times repeated, but on edit button is clicked only when is visible

Waiting does not help, but multiple manual checks for the presence of the button do. Note that the button is clicked just one time (probably) once visible.

Reply:
comment updated.